Doc/c-api/init_config.rst

```diff
@@ -466,7 +466,7 @@ PyConfig
 
    .. c:member:: int dev_mode
 
-      Development mode: see :option:`-X dev <-X>`.
+      If non-zero, enable the :ref:`Python Development Mode <devmode>`.
```

Comment:
```suggestion
      If it has a non-zero value, enable the :ref:`Python Development Mode <devmode>`.
```

Reply:
Actually, the original way matches the other phrasing from above, so I don't think it should be changed.

Reply:
Oh okay, that's fine then.